src/main/java/carRacing/CarRacing.java

```diff
@@ -0,0 +1,52 @@
+package carRacing;
+
+import carRacing.model.Car;
+import carRacing.util.RandomIntUtil;
+import carRacing.view.OutputView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CarRacing {
+    private List<Car> cars;
+    private OutputView outputView;
+
+    public CarRacing(int carCount) {
+        checkIntParam(carCount);
+        this.cars = new ArrayList<>();
+        outputView = new OutputView();
+        enrollCars(carCount);
```

Comment:
CarRacing 생성시 자동차들을 생성하도록 변경 👍 
```suggestion
    public CarRacing(int carCount) {
        checkIntParam(carCount);
        this.cars = enrollCars(carCount);
        this.outputView = new OutputView();
```